db/migrate/20220304195405_migrate_hide_network_preference.rb

```diff
@@ -0,0 +1,35 @@
+class MigrateHideNetworkPreference < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  # Dummy classes, to make migration possible across version changes
+  class Account < ApplicationRecord
+    has_one :user, inverse_of: :account
+    scope :local, -> { where(domain: nil) }
+  end
+
+  class User < ApplicationRecord
+    belongs_to :account
+  end
+
+  def up
+    Setting.where(thing_type: 'User', var: 'hide_network').find_each do |setting|
```

Comment:
This needs to be `Setting.unscoped.where(thing_type: 'User', var: 'hide_network')`, otherwise this won't match any record.

Also calling `Account.reset_column_information` beforehand would ensure the migration works if skipping multiple releases.